src/studio/src/designer/backend/package.json

```diff
@@ -31,12 +31,12 @@
     "jquery-validation-unobtrusive": "^3.2.12",
     "js-beautify": "^1.14.0",
     "json-editor": "^0.7.28",
-    "monaco-editor": "^0.28.1",
+    "monaco-editor": "^0.31.1",
     "popper.js": "^1.16.1",
     "requirejs": "^2.3.6",
     "requirejs-text": "^2.0.16",
     "rivets": "^0.9.6",
-    "select2": "^4.0.13",
+    "select2": "^4.1.0-rc.0",
```

Comment:
Release candidate? Suggest reverting this.

Reply:
Ouch, sorry - missed that.